assets/xml/overlays/ovl_Oceff_Spot.xml

```diff
@@ -0,0 +1,10 @@
+<Root>
+    <File Name="ovl_Oceff_Spot" BaseAddress="0x80972680" RangeStart="0x720" RangeEnd="0xDF8">
+        <Texture Name="sSunSongEffectTex" OutName="sun_song_effect" Format="i8" Width="32" Height="32" Offset="0x720"/>
+        <Array Name="sOceffSpotCylinderVtx" Count="27" Offset="0xB20">
+            <Vtx/>
```

Comment:
It looks like this doesn't get used in the actor, perhaps we should just implicitly extract it instead.